packages/react/src/internal/prepend-base-url.ts

```diff
@@ -0,0 +1,17 @@
+import urljoin from 'url-join';
+
+function isAbsolute(url: string): boolean {
+  return /^https?:\/\//.test(url);
+}
+
+/**
+ * Prepends the given `url` with the given `baseUrl` (can be relative or
```

Comment:
Suggestion: "Prepends the given `baseURL` - which can be relative or absolute - to the `url` , unless the `url` is absolute".